docs/swiper.md

```diff
@@ -0,0 +1,500 @@
+};
+```
+
+---
+
+## Reference
+
+### `vertical`
+
+Swiper vertical layout
+
+|  Type   | Default |
+| :-----: | :-----: |
+| boolean |  false  |
+
+---
+
+### `from`
+
+Initial slide index
+
+|  Type  | Default |
+| :----: | :-----: |
+| number |    0    |
+
+---
+
+### `loop`
+
+Allow loop
+
+|  Type   | Default |
+| :-----: | :-----: |
+| boolean |  false  |
+
+---
+
+### `timeout`
+
+Autoplay slider timeout in secs. Negative value will play reverse
+
+|  Type  | Default |
+| :----: | :-----: |
+| number |    0    |
+
+---
+
+### `gesturesEnabled`
+
+Allow to swipe
+
+|  Type   | Default |
+| :-----: | :-----: |
+| boolean |  true   |
+
+---
+
+### `springConfig`
+
+Tune spring animation on autoplay, touch release or slides changes via buttons.
+
+See official `react-native` [docs](https://facebook.github.io/react-native/docs/animated#spring)
+
+---
+
+### `minDistanceToCapture`
+
+Initiate animation after swipe this distance. It fix gesture collisions inside ScrollView
+
+|  Type  | Default |
+| :----: | :-----: |
+| number |    5    |
+
+---
+
+### `minDistanceForAction`
+
+Minimal part of swiper width (or height for vertical) must be swiped for changing index.
+Otherwise animation restore current slide. Default value 0.2 means that 20% must be swiped for change index
+
+|  Type  | Default |
+| :----: | :-----: |
+| number |   0.2   |
+
+---
+
+### `positionFixed`
+
+Swiper inner container position `fixed` instead `relative`. Fix mobile safari vertical bounce
+
+|  Type   | Default |
+| :-----: | :-----: |
+| boolean |  false  |
+
+---
+
+### `containerStyle`
+
+Outer (root) container style
+
+|      Type      | Default |
+| :------------: | :-----: |
+| object (style) |  none   |
+
+---
+
+### `innerContainerStyle`
+
+Inner container style
+
+|      Type      | Default |
+| :------------: | :-----: |
+| object (style) |  none   |
+
+---
+
+### `swipeAreaStyle`
+
+Swipe area style
+
+|      Type      | Default |
+| :------------: | :-----: |
+| object (style) |  none   |
+
+---
+
+### `slideWrapperStyle`
+
+Each slide wrapper style
+
+|      Type      | Default |
+| :------------: | :-----: |
+| object (style) |  none   |
+
+---
+
+### `controlsEnabled`
+
+Dots and control buttons visible and enabled
+
+|  Type   | Default |
+| :-----: | :-----: |
+| boolean |  true   |
+
+---
+
+### `Controls`
+
+Custom controls component
+
+|   Type    | Default  |
+| :-------: | :------: |
+| component | internal |
+
+---
+
+### `onAnimationStart`
+
+Any swiper animation start
+
+|   Type   | Default |
+| :------: | :-----: |
+| function |  none   |
+
+---
+
+### `onAnimationEnd`
+
+Any swiper animation end
+
+|   Type   | Default |
+| :------: | :-----: |
+| function |  none   |
+
+---
+
+### `onIndexChanged`
+
+Called when active index changed
+
+|   Type   | Default |
+| :------: | :-----: |
+| function |  none   |
+
+---
+
+## `controlsProps`
+
+|   Type   | Default |
+| :------: | :-----: |
+|  object  |  below  |
+
+---
+
+### `cellsStyle`
```

Comment:
Not sure the purpose of these two props `cellStyle` and `cellsContent`

Reply:
`cellsStyle` allows to change placeholder's styles, eg position. `cellsContent` allows to render additional content inside controls placeholders. e.g. rel [this issue](https://github.com/oxyii/react-native-web-swiper/issues/12#issuecomment-520168203)